src/GeoJSON.Text.Test.Benchmark/Deserialize/JsonEmbeddedFileReader.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.IO;
+using System.Reflection;
+
+namespace GeoJSON.Text.Test.Benchmark.Deserialize
+{
+    internal static class JsonEmbeddedFileReader
+    {
+        public static string GetExpectedJson(string fileName)
+        {
+            var myType = typeof(JsonEmbeddedFileReader);
+            var currentNamespace = myType.Namespace;
+
+            var assembly = Assembly.GetCallingAssembly();
+
+            var fileStreamName = $"{currentNamespace}.{fileName}.json";
+
+            Console.WriteLine(fileStreamName);
+
+            using (Stream stream = assembly.GetManifestResourceStream(fileStreamName))
```

Comment:
Should ideally aim for no warnings in build

Reply:
Now throws NullReferenceException if stream is null.